### DISCUSSION TITLE
Oracle Backend

### DISCUSSION THREAD

**Discussion by laktak** [ibis-project/ibis#6556] [Q&A] [answered]
https://ibis-project.org/backends/oracle/ lists the Oracle backend but 

`pip install 'ibis-framework[oracle]'` gives me:

> WARNING: ibis-framework 5.1.0 does not provide the extra 'oracle'

Is Oracle not included in the current release?

**Comment by gforsyth** [comment, marked as answer]
It is not released yet. You can grab a pre release with

```
pip install 'ibis-framework[oracle]' --pre
```

Or you can wait until Monday when we're releasing 6.0.

**Reply by laktak** [reply]
Perfect, thank you!
